ios/RCTContacts/RCTContacts.m

```diff
@@ -40,6 +40,38 @@ - (NSDictionary *)constantsToExport
     }];
 }
 
+RCT_EXPORT_METHOD(getContactsMatchingString:(NSString *)string callback:(RCTResponseSenderBlock) callback)
+{
+    CNContactStore *contactStore = [[CNContactStore alloc] init];
+    if (!contactStore)
+        return;
+    [self getContactsFromAddressBook:contactStore matchingString:string callback:callback];
+}
+
+-(void) getContactsFromAddressBook:(CNContactStore *)store
+                    matchingString:(NSString *)searchString
+                       callback:(RCTResponseSenderBlock)callback
+{
+    NSMutableArray *contacts = [[NSMutableArray alloc] init];
+    NSError *contactError = nil;
+    NSArray *arrayOfContacts = [store unifiedContactsMatchingPredicate:[CNContact predicateForContactsMatchingName:searchString] keysToFetch:@[
+                                                                                                                                               CNContactEmailAddressesKey,
```

Comment:
Solved by above. I usually use the NYTimes style guide, but it really doesn't cover this case...